libs/coin-modules/coin-evm/src/__tests__/coin-tester/docker.ts

```diff
@@ -0,0 +1,42 @@
+import { v2 as compose } from "docker-compose";
+
+const cwd = __dirname;
+
+const delay = (timing: number): Promise<void> =>
+  new Promise(resolve => setTimeout(resolve, timing));
+
+export const spawnAnvil = async (rpc: string): Promise<void> => {
+  console.log("Starting anvil...");
+  await compose.upOne("anvil", {
+    log: true,
+    cwd,
+    env: {
+      ...process.env,
+      RPC: rpc,
+    },
+  });
+
+  const checkAnvilLogs = async (): Promise<void> => {
+    const { out } = await compose.logs("anvil", { cwd });
+
+    if (out.includes("Listening on 0.0.0.0:")) {
+      console.log(" -  ANVIL READY ✅  - ");
```

Comment:
If you don't mind, I'd like to keep some "Chalk" here and there in the logs for important informations